cartography/intel/kandji/devices.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+
+from cartography.intel import crowdstrike
+from cartography.intel.kandji.util import call_kandji_api
+from cartography.util import run_analysis_and_ensure_deps
+from cartography.util import timeit
+
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def _get_devices(kandji_base_uri: str, kandji_token: str) -> List[Dict]:
+    return call_kandji_api("/api/v1/devices", kandji_base_uri, kandji_token)
+
+
+@timeit
+def _load_devices(data: Dict, neo4j_session: neo4j.Session, update_tag: int) -> None:
+    ingest_kandji_device = """
```

Comment:
Instead of writing the query to ingest the data, you can use `cartography.client.core.tx.load()` on your `KandjiDeviceSchema`. Example: https://github.com/lyft/cartography/blob/11b53d5c79eb073c103109ab9edb987be6057eb9/cartography/intel/lastpass/users.py#L68-L81

We should also introduce the idea of a Kandji account or tenant because it's needed for cartography automatic cleanup jobs.